client/src/Components/App/App.js

```diff
@@ -43,7 +43,8 @@ const App = () => {
   }, [feeling]);
 
   useEffect(() => {
-    setUserLogs(entryData);
+    getAllEntries()
+    .then(data => setUserLogs(data));
```

Comment:
Could also use async await here. I know we used a try/catch statement in the server so that could make it more consistent? Not a huge deal though!

Reply:
Good point KB!